pkg/expapi/register.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package expapi
+
+import (
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/latest" // api/latest.init should run first
```

Comment:
api/latest creates a RESTMapper filled with api objects. We need it to initialize before experimental objects are added. I'll expand the comment for clarity.